acstools/tests/test_wfc_findsat_mrt.py

```diff
@@ -0,0 +1,54 @@
+'''
+Test satellite trail detection and masking using findsat_mrt
+'''
+
+from astropy.utils.data import get_pkg_data_filename
+from acstools.findsat_mrt import wfc_wrapper
+from acstools.tests.helpers import BaseACSTOOLS
+import os
+from astropy.io.fits import FITSDiff
+import logging
+
+logging.basicConfig()
+LOG = logging.getLogger()
+LOG.setLevel(logging.INFO)
+
+
+class TestFindsatMRT(BaseACSTOOLS):
+
+    detector = 'wfc'
+
+    def test_wfc_wrapper(self, tmpdir):
```

Comment:
If you want to future-proof, `tmpdir` is now considered "legacy" and `tmp_path` should be used instead. See https://docs.pytest.org/en/stable/how-to/tmp_path.html#tmpdir-and-tmpdir-factory

Reply:
Response: I have replaced tmpdir with tmp_path